torch/csrc/jit/mobile/model_tracer/OperatorCallTracer.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <ATen/record_function.h>
+
+namespace torch {
+namespace jit {
+namespace mobile {
+/* The OperatorCallTracer class handles the attachment and removal of a
+ * recording callback that traces invocation of ATen (and other) PyTorch
+ * operators that get called via the Dispatcher.
+ *
+ * You can get the set of operators that were called (op_name.overload_name)
+ * using getCalledOperators().
+ *
+ * Note: This class is not thread safe or re-entrant, and should not be used
+ * across multiple threads of execution.
+ *
+ */
+struct OperatorCallTracer final {
+  static std::set<std::string> called_operators_;
```

Comment:
Does this set really needs to be ordered? I.e. wouldn't `std::unordered_set` be faster?

Reply:
Its expected for these traced operators to be serialized so its convenient for the ordering to always be consistent. The efficiency difference should be small, the majority of models in my experience have ~100 operators.